Src/BootCamp.Chapter/Computer/Gpu.cs

```diff
@@ -1,6 +1,52 @@
 ﻿namespace BootCamp.Chapter.Computer
 {
-    public class Gpu
+    public class Gpu : Component
     {
+        private string _chipsetManufacturer;
+        public string GetChipsetManufacturer()
+        {
+            return _chipsetManufacturer;
+        }
+
+        private GPUMemoryType _memoryType;
+        public GPUMemoryType GetMemoryType()
+        {
+            return _memoryType;
+        }
+
+        private int _memorySize;
+        public int GetMemorySize()
+        {
+            return _memorySize;
+        }
+
+        private int _memoryClock;
+        public int GetMemoryClock()
+        {
+            return _memoryClock;
+        }
+
+        private int _gpuClock;
+        public int GetGpuClock()
+        {
+            return _gpuClock;
+        }
+
+
+        public Gpu(string id, string manufacturer, string modelName, string chipsetManufacturer, GPUMemoryType memoryType, int memorySize, int memoryClock, int gpuClock) : base (id, manufacturer, modelName)
+        {
+            _chipsetManufacturer = chipsetManufacturer;
+            _memoryType = memoryType;
+            _memorySize = memorySize;
+            _memoryClock = memoryClock;
+            _gpuClock = gpuClock;
+        }
+    }
+
+    public enum GPUMemoryType
+    {
```

Comment:
Also, are enums fine in this case or would you use something else here? My main goal was to avoid having to guess for memory types. Similar stuff to this is implemented at other components as well.

I guess in a real app, you'd have a database or something that would store these right?